include/pm3_cmd.h

```diff
@@ -536,6 +537,15 @@ typedef struct {
 /* Set if this device understands the chip info command */
 #define DEVICE_INFO_FLAG_UNDERSTANDS_CHIP_INFO       (1<<5)
 
+/* Set if this device understands the version command */
+#define DEVICE_INFO_FLAG_UNDERSTANDS_VERSION         (1<<6)
+
+// Different versions here. Each version should increse the number
+#define BL_VERSION_INVALID  0
+#define BL_VERSION_1_0_0    1
```

Comment:
I'm a bit puzzled between:
* the macro, using a major_minor_patch representation
* the actual value, not meant to be parsed as maj/min/patch (I'd expect sth like  1<<16+0<<8+0), maybe even just a BL_MAKE_VERSION(maj,min,patch) macro, see e.g. https://vulkan.lunarg.com/doc/view/1.0.30.0/linux/vkspec.chunked/apds03.html
* the BL_VERSION_FIRST/LAST where I would have expected a simple check on the major, such as BL_VERSION_MAJOR(version)==X